src/interface/raftex.thrift

```diff
@@ -133,6 +133,7 @@ struct GetStateResponse {
     6: LogID            last_log_id;
     7: TermID           last_log_term;
     8: Status           status;
+    9: list<binary>     peers;
```

Comment:
`peers` have been used?

Reply:
what used mean?

Reply:
where is the use of it

Reply:
monitor tools

Reply:
Maybe you could add some explanation in the future pr

Reply:
> 

It's very important for us to retrieve raft peers info when trouble shooting some issue.
BTW, think we should also including this info in storage monitring metrics.